go/ql/lib/change-notes/2023-18-09--add-support-for-fasthttp-framework.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Support for the [Fasthttp framework](https://github.com/valyala/fasthttp/) has been added.
```

Comment:
The format for the change notes is different (see below). Additionally, the double dash in the filename is not required and the date format is YYYY-MM-DD while you seem to have used YYYY-DD-MM.

```suggestion
---
category: minorAnalysis
---
* Support for the [Fasthttp framework](https://github.com/valyala/fasthttp/) has been added.
```